lang/cs/Org.Apache.REEF.IMRU/API/IMRUJobDefinitionBuilder.cs

```diff
@@ -203,6 +211,17 @@ public IMRUJobDefinitionBuilder SetUpdateTaskMemory(int memory)
         }
 
         /// <summary>
+        /// Sets Per Map Configuration
```

Comment:
This is quite confusing, as this is used to set the configuration used to instantiate the `IPerMapConfiguration` on the Driver, which in turn returns the configurations used for the mappers, right?

Also, why not make this part of a `ConfigurationModule` which has a `OptionalImpl<IPerMapConfiguration>`? That could then be bound to a `Name<Set<IPerMapConfiguration>>` to allow for future expansion.